package.json

```diff
@@ -29,7 +29,7 @@
   "license": "ISC",
   "dependencies": {
     "@sentry/node": "^7.51.0",
-    "chainsauce": "github:gitcoinco/chainsauce",
+    "chainsauce": "github:gitcoinco/chainsauce#49bee375cc230097ef024c0f7107c13393f5c2ac",
```

Comment:
This is not really needed as the hash is already saved in the lockfile, i.e. there is no danger of installing a version other than the correct one for a given commit.